modules/core/04-channel/v2/keeper/grpc_query_test.go

```diff
@@ -347,31 +344,21 @@ func (suite *KeeperTestSuite) TestQueryPacketCommitment() {
 			"invalid sequence",
 			func() {
 				req = &types.QueryPacketCommitmentRequest{
-					ChannelId: ibctesting.FirstChannelID,
-					Sequence:  0,
+					ClientId: ibctesting.FirstClientID,
+					Sequence: 0,
 				}
 			},
 			status.Error(codes.InvalidArgument, "packet sequence cannot be 0"),
 		},
-		{
-			"channel not found",
```

Comment:
Removed these cases. They still error but now equivalent to the queried key not existing